Examples/Neutral_density_calculation.ipynb

```diff
@@ -0,0 +1,1066 @@
+{
```

Comment:
<p>how about replacing</p>

<pre class="ql-syntax" spellcheck="false">&nbsp; &nbsp; if model == 'mom5':
&nbsp; &nbsp; &nbsp; &nbsp; insitu_temp = gsw.t_from_CT(absolute_salinity, temp - 273.15, pressure).rename('temp')
&nbsp; &nbsp; elif model == 'mom6':
&nbsp; &nbsp; &nbsp; &nbsp; conservative_temperature = gsw.CT_from_pt(absolute_salinity, temp)
&nbsp; &nbsp; &nbsp; &nbsp; insitu_temp = gsw.t_from_CT(absolute_salinity, conservative_temperature, pressure).rename('temp')
</pre><p>with</p>

<pre class="ql-syntax" spellcheck="false">&nbsp; &nbsp; if model == 'mom5':
        conservative_temp_in_C = temp - 273.15
&nbsp; &nbsp; elif model == 'mom6':
&nbsp; &nbsp; &nbsp; &nbsp; conservative_temp_in_C = gsw.CT_from_pt(absolute_salinity, temp)
&nbsp; &nbsp; 
    insitu_temp = gsw.t_from_CT(absolute_salinity, conservative_temp_in_C, pressure).rename('temp')
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/417/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
I did this